trunk/SUAVE/Components/Energy/Converters/Combustor.py

```diff
@@ -112,28 +123,132 @@ def compute(self,conditions):
         ht4     = Cp*Tt4
         ho      = Cp*To
         ht_in   = Cp*Tt_in
-
+        
         # Using the Turbine exit temperature, the fuel properties and freestream temperature to compute the fuel to air ratio f
         f       = (ht4 - ht_in)/(eta_b*htf-ht4)
 
         # Computing the exit static and stagnation conditions
         ht_out  = Cp*Tt4
-        Pt_out  = Pt_in*pib
         
         # pack computed quantities into outputs
         self.outputs.stagnation_temperature  = Tt4
         self.outputs.stagnation_pressure     = Pt_out
         self.outputs.stagnation_enthalpy     = ht_out
         self.outputs.fuel_to_air_ratio       = f 
     
-    
-    
+    def compute_rayleigh(self,conditions):
+        """ This computes the output values from the input values according to
+        equations from the source.
+
+        Assumptions:
+        Constant efficiency and pressure ratio
+
+        Source:
+        https://web.stanford.edu/~cantwell/AA283_Course_Material/AA283_Course_Notes/
+
+        Inputs:
+        conditions.freestream.
+          isentropic_expansion_factor         [-]
+          specific_heat_at_constant_pressure  [J/(kg K)]
+          temperature                         [K]
+          stagnation_temperature              [K]
+        self.inputs.
+          stagnation_temperature              [K]
+          stagnation_pressure                 [Pa]
+
+        Outputs:
+        self.outputs.
+          stagnation_temperature              [K]  
+          stagnation_pressure                 [Pa]
+          stagnation_enthalpy                 [J/kg]
+          fuel_to_air_ratio                   [-]
+
+        Properties Used:
+        self.
+          turbine_inlet_temperature           [K]
+          pressure_ratio                      [-]
+          efficiency                          [-]
+          area_ratio                          [-]
+          fuel_data.specific_energy           [J/kg]
+        """         
+        # unpack the values
+
+        # unpacking the values from conditions
+        gamma  = conditions.freestream.isentropic_expansion_factor 
+        Cp     = conditions.freestream.specific_heat_at_constant_pressure
+        To     = conditions.freestream.temperature
+        Tto    = conditions.freestream.stagnation_temperature
+        
+        # unpacking the values form inputs
+        Tt_in  = self.inputs.stagnation_temperature
+        Pt_in  = self.inputs.stagnation_pressure
+        Mach   = self.inputs.mach_number
+        Tt4    = self.turbine_inlet_temperature
+        pib    = self.pressure_ratio
+        eta_b  = self.efficiency
+        
+        # unpacking values from self
+        htf    = self.fuel_data.specific_energy
+        ar     = self.area_ratio
+        
+        # Rayleigh flow analysis, constant pressure burner
+            
+        # Initialize arrays
+        M_out  = 1*Pt_in/Pt_in
+        Ptr    = 1*Pt_in/Pt_in
```

Comment:
Can clean up extra white space here through the end of the script

Reply:
fixed